README.md

```diff
@@ -857,9 +857,7 @@ This repository also provides one such platforms where contributers come over an
 | [UNO game with computer](https://github.com/kunjgit/GameZone/tree/main/Games/UNO_game_with_Computer) |
 |[Dice_Rolling_Simulator](https://github.com/priyashuu/GameZone/tree/main/Games/Dice_rolling_simulator)|
 |[Space_Dominators](https://github.com/kunjgit/GameZone/tree/main/Games/Space_Dominators)|
-| [Simon_Says](https://github.com/kunjgit/GameZone/tree/main/Games/Simon_Says) |
-|[Idle_miner](https://github.com/kunjgit/GameZone/tree/main/Games/Idle_miner)|
-|[Five_Nights_at_Freddys](https://github.com/kunjgit/GameZone/tree/main/Games/Five_Nights_at_Freddys)|
+|[Matching_Pair](https://github.com/priyashuu/GameZone/tree/main/Games/Matching_pair)
```

Comment:
change the your id name with kunjgit to make changes visible on the site as other entries do